Deployment/purgeOldReportData.sh

```diff
@@ -13,10 +13,10 @@ echo "=====> Older Data is from date : $past_date"
 
 echo "=====> $current_date: INFO: Scanning reports Data for retention.."
 
-echo "=====> INFO: Purging reports older than $past_date from $report_path....."
+echo "=====> INFO: Purging reports older than $past_date from $activeUserReport_path....."
```

Comment:
> purgeOldReportData.sh

the procedure is related to retention management.
please use this fact for naming more abstract "purge old data".

Reply:
provided that there are going to be many different reports  (which is a fact that I've stated multiple times), could  you plan for extension beforehand? i.e. please make retention management logic reusable for any dataset and for any period?

Reply:
I have already updated the file name.. This is an outdated file.. New name is purgeOldActiveUsersReportData.sh
And I was thinking of making different scripts for different reports.. Do you want me to create one single script which will do data retention for all kind of reports ?

Reply:
I have updated the script to be used as a generic script for any kind of data retention. Script name has been updated accordingly.